specification/under_development/profiles/Base_Profile.md

```diff
@@ -0,0 +1,262 @@
+pointers (deref or not). that info is captured in the target triple/duo.
+
+Qubits may not be used after they have been measured. Qubits and results need to
+be numbered consecutively, starting at 0. No qpu support needed for measuring
+individual qubits, but backend would support output selection?
+
+## Quantum Instruction Set
+
+For a Quantum Instruction Set to be compatible with the Base Profile, it needs
+to satisfy the following requirements:
+
+- Since the Base Profile doesn't permit to define local variables, all
+  instructions are required to return void.
+
+## Output Recording
+
+Log format is a separate spec. What the i8* can be is a separate spec. Default
+spec for front-ends to compile into. TODO: do string labels need to be zero
+terminated?
+
+The following functions are declared and used to record the program output: |
+Function                  | Signature         | Description |
+|---------------------------|-------------------|-------------| |
+__quantum__rt__initialize_record_output    | `void(i8*)`  | Inserts a marker in
+the output log that contains an identifier for the used labeling scheme. The
+backend may choose which output format to use, and the label identifier is
+omitted for output formats that do not support labeling. | |
+__quantum__rt__tuple_record_output    | `void(i64, i8*)`  | Inserts a marker in
+the output log that indicates the start of a tuple and how many tuple elements
+are going to be logged. The second parameter reflects an label for the tuple.
+The backend may choose which output format to use. Depending on the used format,
+the label will be logged or omitted. | | __quantum__rt__array_record_output    |
+`void(i64, i8*)`  | Inserts a marker in the output log that indicates the start
+of an array and how many array elements are going to be logged. The second
+parameter reflects an label for the array. The backend may choose which output
+format to use. Depending on the used format, the label will be logged or
+omitted. | |__quantum__rt__result_record_output   | `void(%Result*, i8*)`  |
+Adds a measurement result to the output log. The second parameter reflects an
+label for the result value. The backend may choose which output format to use.
+Depending on the used format, the label will be logged or omitted. |
+
+TODO: It is sufficient to use the same functions for output recording
+independent on the output format; i.e. the output format does not need to be
+reflected in the IR, and it is sufficient to label the format it in the output
+itself. The output itself then needs to contain both the output format
+identifier (defined by the backend), as well as an identifier for the labeling
+scheme (as defined in the program IR itself). -> for base profile, no
+computations (classical or quantum, including calls to rt functions other than
+record_output* functions) can be performed after the call to
+__quantum__rt__initialize_record_output
```

Comment:
I'm not sure there has been sufficient discussion of classical vs quantum computations and there separation so far in the base profile?

Reply:
The notes that this comment applied to have been removed and replaced by hopefully comprehensive text. I think your comment might be addressed. There is currently only one mention of classical computations, and it's at the beginning of the Data Types and Values section. Please take a look and confirm if this addressed your concern.